CoreMapView/src/au/gov/asd/tac/constellation/views/mapview/overlays/InfoOverlay.java

```diff
@@ -288,8 +288,8 @@ public void onManipulation(final MapEvent mapEvent) {
             scopedListenersList.forEach(scopedListeners -> {
                 if (scopedListeners.isInScope(mapEvent)) {
                     scopedListeners.listeners.forEach(listener -> {
-                        if (listener instanceof UnfoldingMap) {
-                            final UnfoldingMap listeningMap = (UnfoldingMap) listener;
+                        if (listener instanceof UnfoldingMap unfoldingMap) {
+                            final UnfoldingMap listeningMap = unfoldingMap;
```

Comment:
the 2nd line shouldn't be needed now that the variable has been assigned in the instanceof pattern match (you could though change `unfoldingMap` declaration to `listeningMap` and then you don't have to change any further code)